controllers/openstackcluster_controller_test.go

```diff
@@ -160,9 +170,12 @@ var _ = Describe("OpenStackCluster controller", func() {
 		Expect(err).To(BeNil())
 		req := createRequestFromOSCluster(testCluster)
 
+		clientCreateErr := fmt.Errorf("Test failure")
+		mockScopeFactory.SetClientScopeCreateError(clientCreateErr)
+
 		result, err := reconciler.Reconcile(ctx, req)
 		// Expect error for getting OS clinet and empty result
```

Comment:
```diff
- // Expect error for getting OS clinet and empty result
+ // Expect error for getting OS client and empty result
```

Reply:
I'm inclined to leave this out of this patch as it's not strictly related. This PR has enough tentacles already!